main.py

```diff
@@ -112,22 +114,53 @@ def markers(methods=["GET", "POST"]):
         marker.put()
         return make_response(json.dumps(marker.serialize(self.user)))
 
+#TODO: if not in future use - remove
```

Comment:
You're right, we currently don't support insertion and deletion of markers, so it can be removed.